scipy/signal/spline.py

```diff
@@ -6,7 +6,8 @@
 
 from . import _spline
 
-__all__ = ['cspline2d', 'qspline2d', 'sepfir2d', 'symiirorder1', 'symiirorder2']
+__all__ = ['cspline2d', 'qspline2d', 'sepfir2d', 'symiirorder1',  # noqa: F822
+           'symiirorder2']  # noqa: F822
```

Comment:
```suggestion
__all__ = ['cspline2d', 'qspline2d', 'sepfir2d', 'symiirorder1',  # noqa: F822
           'symiirorder2'] 
```
The second `# noqa` isn't technically required here (I thought it was the case but had to try it to be sure).